cdap-docs/developer-manual/source/cloud-runtimes/provisioners/aws-emr.rst

```diff
@@ -8,5 +8,68 @@
 Amazon Elastic MapReduce
 ========================
 
-Describe what EMR is, then describe each config setting
+Amazon EMR provides a managed Hadoop framework that makes it easy, fast, and cost effective
```

Comment:
This seem likes copied from Amazon EMR page. I am not sure if that is fine or not so it will be nice to check.